modules/apps/headless-apio/content-space/content-space-apio-api/src/main/java/com/liferay/content/space/apio/architect/model/ContentSpace.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.content.space.apio.architect.model;
+
+import com.liferay.apio.architect.annotation.Id;
+import com.liferay.apio.architect.annotation.Vocabulary;
+import com.liferay.apio.architect.identifier.Identifier;
+import com.liferay.folder.apio.architect.identifier.RootFolderIdentifier;
+import com.liferay.person.apio.architect.identifier.PersonIdentifier;
+
+import java.util.List;
+import java.util.Locale;
+
+/**
+ * Holds information about a content space. The internal method {@code
+ * com.liferay.portal.kernel.model.GroupModel#getGroupId()} returns the
+ * identifier.
+ *
+ * @author Javier Gamarra
+ * @author Cristina González
+ */
+@Vocabulary.Type("ContentSpace")
+public interface ContentSpace extends Identifier<Long> {
```

Comment:
mmm... what we thought to minimize renames and package changes was having this code (the old representor) in the identifier (that way we wouldn't have to change any other API that the one we were migrating) and then, when all the APIs are migrated, rename the identifier and change all packages at the same time. It's ok in this case because no-one is using ContentSpaceIdentifier but if someone is using it, relations are not going to work with the deprecated version.